sage-ios/SAGE/SAGE/RequestHoursView.swift

```diff
@@ -71,12 +75,14 @@ class RequestHoursView: UIView {
         self.dateFormatter.dateStyle = .NoStyle
         self.dateFormatter.timeStyle = .ShortStyle
```

Comment:
because dateformatter is a private variable. shouldnt be modified by any other object, and these are just default vals